ui/src/app/views/account-creation-success/component.html

```diff
@@ -12,4 +12,10 @@ <h4>Now you need to log in using this account</h4>
   <div class="row" style="padding-top:0.5rem;">
     <button class="btn btn-primary" (click)="signIn()">LOGIN</button>
   </div>
+  <div class="row" style="padding-top:1rem;">
+    <h4>Please add a layer of protection with 2-step Verification</h4>
```

Comment:
Workbench requires 2-step verification: you have three days to enable 2-step verification on your account.